docs/notebooks/inequality_constraints.pct.py

```diff
@@ -239,13 +239,14 @@ def batch_efi(at):
 plt.show()
 
 # %% [markdown]
-# Finally, we compare the resulting regret with a non-batch strategy.
+# Finally, we compare the regret from the non-batch strategy (left panel) to the regret from the batch strategy (right panel).
+# In the following plots each marker represents a query point. The x-axis is the index of the query point (where the first queried point has index 0), and the y-axis is the observed value. The vertical blue line denotes the end of initialisation/start of optimisation. Green points satisfy the constraint, red points do not.
 
 # %%
 from util.plotting import plot_regret
 
-mask_fail = constraint_data.observations.numpy() >= Sim.threshold
-batch_mask_fail = batch_constraint_data.observations.numpy() >= Sim.threshold
+mask_fail = constraint_data.observations.numpy() > Sim.threshold
+batch_mask_fail = batch_constraint_data.observations.numpy() > Sim.threshold
```

Comment:
@vpicheny is this right?

Reply:
yes, fail is when > threshold.

Reply:
it propagates to plot regret in that way:
`safe_obs_values[mask_fail] = np.max(obs_values)`

so when mask = 1, ignore the value for the regret plot